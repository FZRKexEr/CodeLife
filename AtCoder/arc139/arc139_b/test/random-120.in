100
95 3 6 19 14 20
74 11 19 18 19 12
38 18 3 19 3 8
14 18 2 2 4 13
92 1 9 18 20 20
44 9 12 11 18 5
3 4 18 7 20 9
60 20 15 3 12 8
70 3 18 1 8 7
48 1 1 16 19 2
33 3 15 1 1 4
10 20 14 18 3 10
79 4 15 12 18 7
32 1 2 19 5 17
71 19 6 17 6 18
41 1 17 4 2 10
58 9 16 12 5 8
38 10 3 4 1 18
41 5 2 13 17 1
41 12 13 10 12 16
75 5 1 13 8 12
4 6 15 1 8 2
6 7 15 20 19 5
54 7 5 8 16 6
96 5 14 5 3 12
4 20 16 12 14 7
40 18 5 18 16 9
97 3 15 19 12 1
23 9 3 19 4 20
18 1 8 7 16 11
85 16 12 20 1 11
13 9 19 8 14 2
78 17 18 17 5 7
16 12 13 17 20 14
83 7 2 7 16 3
33 3 11 11 11 7
50 3 11 7 13 10
2 2 9 4 1 1
75 18 8 8 15 18
97 19 19 19 3 1
66 7 17 8 9 7
12 9 12 17 7 14
89 7 15 7 13 6
21 15 2 8 3 13
11 6 4 17 16 7
76 20 2 9 14 7
26 11 8 1 4 13
60 15 4 9 1 9
23 1 20 12 8 20
15 14 14 16 11 1
34 16 6 15 20 6
5 11 7 15 16 11
84 20 19 5 2 17
21 3 14 3 8 5
27 7 5 19 16 9
14 10 20 18 9 6
12 19 6 17 4 3
97 1 20 10 10 18
76 3 8 1 18 2
68 17 18 10 10 18
69 18 17 13 8 17
46 8 11 19 5 16
47 2 18 11 8 2
90 15 20 6 8 2
12 15 15 20 9 4
37 4 4 6 20 13
18 17 18 13 6 17
5 1 12 14 5 1
57 4 10 8 4 18
99 4 12 5 9 20
41 9 10 7 20 3
79 12 13 9 16 1
89 5 4 1 12 10
79 20 9 4 16 6
94 6 14 13 5 8
61 14 9 10 13 6
98 16 14 15 3 9
79 7 10 1 15 18
58 19 12 19 6 6
47 8 13 4 9 2
63 18 15 5 16 10
22 10 8 3 5 3
92 11 9 17 7 7
72 13 17 18 2 19
12 11 8 17 14 4
51 8 18 2 20 12
82 10 3 10 16 18
11 3 1 10 19 17
22 5 3 20 18 8
5 14 11 10 14 9
84 20 18 19 1 12
41 3 6 17 12 16
66 3 8 17 4 2
33 14 17 5 11 13
50 9 3 7 20 19
3 4 11 17 7 5
16 7 3 13 2 9
15 17 1 2 19 5
26 2 3 4 5 2
10 19 3 3 7 19
