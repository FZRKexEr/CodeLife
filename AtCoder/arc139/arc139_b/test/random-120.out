352
107
44
28
236
68
21
32
37
96
9
180
53
160
88
34
41
35
33
68
120
4
120
80
62
48
72
49
66
36
56
46
130
65
130
21
66
1
84
43
47
14
56
42
40
98
12
4
44
17
38
75
28
24
52
81
6
242
22
40
59
47
55
22
240
123
17
25
64
175
19
15
89
64
60
61
48
79
37
15
46
16
70
81
31
54
148
77
68
50
80
142
36
34
133
51
29
30
21
30
